models/public/erfnet/README.md

```diff
@@ -0,0 +1,114 @@
+# erfnet
+
+## Use Case and High-Level Description
+
+  This is a ONNX\* version of `erfnet` model designed to perform real-time lane detection.
+  This model is pre-trained in PyTorch* framework and retrained by CULane.
+  For details see [repository](https://github.com/Zhangxianwen2021/ERFNet),
+  paper of [ERFNet](https://doi.org/10.1109/TITS.2017.2750080) and [repository](https://github.com/cardwing/Codes-for-Lane-Detection/tree/master/ERFNet-CULane-PyTorch)
+
+## Specification
+
+| Metric            |        Value          |
+|-------------------|-----------------------|
+| Type              | Semantic segmentation |
+| GFLOPs            | 11.13                 |
+| MParams           | 7.87                  |
+| Source framework  | PyTorch\*             |
+
+## Accuracy
+
+|  Metric  |  Value |
+|  ------  | -------|
+| mean_iou | 76.47% |
+
+## Input
+
+### Original model
+
+Image, name - `input_1`, shape - `1,3,208,976`, format is `B,C,H,W` where:
+
+- `B` - batch size
+- `C` - channel
+- `H` - height
+- `W` - width
+
+
+Channel order is `BGR`.
+
+### Converted model
+
+Image, name - `input_1`, shape - `1,3,208,976`, format is `B,C,H,W` where:
+
+- `B` - batch size
+- `C` - channel
+- `H` - height
+- `W` - width
+
+
+Channel order is `BGR`.
+
+## Output
+
+### Original model
+Feature map, name - `output1`, shape - `1,5,208,976`, format is `B,C,H,W` where:
```

Comment:
please provide more info about the output format (what each class in the channel dim means)

Reply:
or at least which labels should be used if you do not use all 5.
Also, please provide txt file with labels for running demo to this folder:
https://github.com/openvinotoolkit/open_model_zoo/tree/master/data/dataset_classes

Reply:
When running the convert demo, we don't use the txt file.
When accuracy-check demo, we use the label json file and the file has been provided in downloaded dataset.
Do I need to provide the txt file?

Reply:
you can run demo with file, because it is optional argument, from user perspective it is very important to know which classes model return. At least explain it in documentation. I suggested to add file only in case, if it is difficult to you to provide this information in text form (this file can be just referenced in readme).

The most strange thing that model output labels dim is 5, while in AC labels you put only 2, so I think this detail is extrimally important for this case, otherwise it my confuse omz users

Reply:
Probably, it still requires more information. If I correctly understand, dataset format, model can be used for line detection on multi-lane road (maximum number of lanes - 4) and each class in feature map represent lane for specific place. Could you extend output description with such information? Now, it still looks confusing for general omz users without this specific task context